textproc/help2man/Portfile

```diff
@@ -41,8 +41,11 @@ configure.universal_args-delete --disable-dependency-tracking
 
 variant universal {}
 
-post-patch {
-    reinplace "s|-shared|-bundle ${configure.ldflags} [get_canonical_archflags ld]|" ${worksrcpath}/Makefile.in
+platform darwin {
+    post-patch {
+        reinplace "s|-shared|-bundle ${configure.ldflags} [get_canonical_archflags ld]|" \
```

Comment:
But even on non-Darwin operating systems we would want to use the right LDFLAGS.

Reply:
Fair enough, `${configure.ldflags}` is to be kept then.